server/controller/userController.js

```diff
@@ -0,0 +1,30 @@
+const TimeEntry = require('../models/timeEntry')
+
+exports.getTimeEntries = async(req, res) => {
+  try {
+    let entries = await TimeEntry.find();
+    // successful
+    res.status(200).json(entries);
+  }
+  catch (err) {
+    //server error
+    res.status(500).json({
+      message: err.message
+    })
+  }
+}
+exports.setTimeEntry = async(req,res)=>{
+  let data = req.body;
+  let entry = new TimeEntry(data);
+  try {
+    entry.save();
+    // created successful
+    res.status(201).json(entry);
```

Comment:
can we use res.send(entry) ?

instead of sending 201 status here

Reply:
yes, we can but in that case, express will implicitly send 200 status.
Here I have used 201 because it tells something is created successfully.